picasso/src/main/java/com/squareup/picasso/RequestCreator.java

```diff
@@ -426,7 +426,10 @@ public Bitmap get() throws IOException {
     String key = createKey(finalData, new StringBuilder());
 
     Action action = new GetAction(picasso, finalData, memoryPolicy, networkPolicy, tag, key);
-    return forRequest(picasso, picasso.dispatcher, picasso.cache, picasso.stats, action).hunt();
+    Bitmap result =
+        forRequest(picasso, picasso.dispatcher, picasso.cache, picasso.stats, action).hunt();
+    picasso.insertIntoMemoryCache(key, result);
```

Comment:
this doesn't respect the caching strategy.  maybe revert `insertIntoMemoryCache` and instead do:
```
   if (shouldWriteToMemoryCache(action.getMemoryPolicy())) {
     picasso.cache.set(key, result);
   }
```
WDYT?

Reply:
yes, sounds good.
also, i forgot to update the doc on `get()`.